include/hal/library/cryptlib.h

```diff
@@ -1740,6 +1740,25 @@ bool libspdm_x509_get_issuer_name(const uint8_t *cert, size_t cert_size,
                                   uint8_t *cert_issuer,
                                   size_t *issuer_size);
 
+/**
+ * Retrieve the public key encrypt algo oid from one X.509 certificate.
+ *
+ * @param[in]      cert                 Pointer to the DER-encoded X509 certificate.
+ * @param[in]      cert_size            size of the X509 certificate in bytes.
+ * @param[in,out]  cert_public_key_raw  the public key raw data in cert, only rsa public key will output
```

Comment:
This is only for RSA public keys?

Reply:
Thanks. I have changed the comment. This is only for RSA and ECC key. 
In OpenSSL, the `public_key_size = i2d_PublicKey(public_key, NULL);` will be failed for ED448/ED25519.
In MbedTLS, it only support RSA and ECC now.